### ISSUE TITLE
some problrm of the edited image

### ISSUE THREAD

**Issue by DanMerry** [Mingzhen-Huang/D-TIIL#2]
Excuse me, we found some problems in the reproduction process, and the edited image is not as realistic as the paper shows. Even though the model_fine_tuning_optimization_steps is set to 500
# prompt
prompt = 'Wolf shown recovering will be adopted by a veterinary technician'
# original image
![example](https://github.com/Mingzhen-Huang/D-TIIL/assets/52539280/e3dc366f-1747-43a6-bea9-1f0e6ec62c51)
# edited image
![first_edit_img](https://github.com/Mingzhen-Huang/D-TIIL/assets/52539280/390afb2b-390a-4f13-9cc2-e0373358cad3)
The edited image is far from the original, is there any parameter need to be adjusted?
# final mask
![final_mask_img](https://github.com/Mingzhen-Huang/D-TIIL/assets/52539280/3f073829-ca0d-4b0e-b7c4-2444954dc720)
The final mask can not convey the ground truth of the region.

**Comment by jialingYK** [comment]
Hi, thank you for bringing this to our attention.
Our method is quite sensitive to hyper-parameters. I recommend trying the following settings:

diffusion_model_learning_rate=4e-6, embedding_learning_rate=0.001, model_fine_tuning_optimization_steps=200, text_embedding_optimization_steps=500, threshold=0.17


### **Prompt**
prompt = 'A wolf shown recovering will be adopted by a veterinary technician'

### **Original image**
![example](https://github.com/Mingzhen-Huang/D-TIIL/assets/79334800/c9e6ff4e-ea8d-4f4e-ab6a-a8868e8217ae)

### **Edited image**
![edited_image](https://github.com/Mingzhen-Huang/D-TIIL/assets/79334800/bd76f266-e07f-4d44-80cd-57c373e0434b)


### **Final mask**
![blended_image_with_max_mask](https://github.com/Mingzhen-Huang/D-TIIL/assets/79334800/2ef12362-5471-46ed-b2f0-bb36ed8e37e1)

**Comment by DanMerry** [comment]
Thanks for your response, the effect is awsome!!!
